GOVERNANCE.MD

```diff
@@ -96,4 +111,10 @@ By contributing, you agree that your contributions will be licensed under the sa
 
 ---
 
+## 8. Communication
```

Comment:
Do you want to add the github discussions here too, or should we handle that separately?

Reply:
Actually that's a great idea.